src/components/SaveEventButton.test.js

```diff
@@ -3,13 +3,31 @@ import React from "react";
 import { Animated, Easing } from "react-native";
 import { shallow } from "enzyme";
 import SaveEventButton from "./SaveEventButton";
+import Touchable from "./Touchable";
+import text from "../constants/text";
 
 it("renders correctly", () => {
   const output = shallow(<SaveEventButton active={false} onPress={() => {}} />);
   output.setState({ progress: new Animated.Value(0) });
   expect(output).toMatchSnapshot();
 });
 
+it("has correct accessibilityLabel when active=false", () => {
+  const output = shallow(<SaveEventButton active={false} onPress={() => {}} />);
+  output.setState({ progress: new Animated.Value(0) });
+  expect(output.find(Touchable).props().accessibilityLabel).toEqual(
```

Comment:
I would use a snapshot test instead. You can simplify these 3 tests (including  `it("renders correctly", () => {` in to just two. One for active, one for inactive.

Reply:
This is kind of the point of snapshot tests. What is useful is when others make changes in future. You can see when they update the snapshot the assertion has changed. Leaving both statements in means the test needs to be updated in two places if the copy changes.

Reply:
ok, both are just snapshot tests now. I've seen both approaches to snapshot tests - just snapshot tests, snapshot tests alongside specific property tests, they both have pros and cons. Not too worried about it for this test :)